wiki/cli.py

```diff
@@ -26,14 +26,19 @@ def main(ctx, directory):
 
 @main.command()
 @click.option('--debug/--no-debug', envvar='WIKI_DEBUG', default=False)
+@click.option('--host', envvar='HOST', default=None)
+@click.option('--port', envvar='PORT', default=None, type=int)
```

Comment:
this should be named `WIKI_PORT` to avoid conflicts with other programs and keep consistency with `WIKI_DEBUG`